src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import javafx.application.Application;
+
+/**
+ * The Launcher class serves as the entry point for the Duke application.
+ * It uses JavaFX's Application.launch method to launch the application by
+ * starting the Main class.
+ * This class is primarily used to work around classpath issues that can
+ * occur when launching JavaFX applications.
+ */
```

Comment:
I am not sure if I like how lengthy the JavaDoc is. Perhaps you could make it more concise to 1-2 sentences.